api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -87,6 +87,30 @@ message HttpConnectionManager {
       EGRESS = 1;
     }
 
+    // Describes custom tags for the active span.
```

Comment:
Yes lets please de-dup this message and put it in a single place. I think a new tracing.proto in the types directory would be fine? Or feel free to suggest something else.

Reply:
fixed